### PR TITLE
Add tables for tracking RTP job IDs and multiple obsid tasks

### PR DESCRIPTION
**Pull request by bhazelton**
@jburba @plaplant This PR has the structure ~(but not the testing yet)~ that we talked about. It should give you the API to build against, let me know if there's anything you'd like to see change or added.

~It's branched off the `corr-config` branch, so it has all those change in as well. When PR #518 is merged, I'll rebase this and it'll only have the new changes. For now, focus on the changes in `rtp.py` and the methods in `mc_session.py` that have `rtp` in the name. Or you can just look at the changes in the very last commit.~

### CODE REVIEW

**Comment by bhazelton** [comment]
Still todo:
- Update documentation (tex file)
- add scripts to add to these 4 new tables

File: hera_mc/tests/test_rtp.py [hunk 1]
```diff
@@ -177,14 +291,25 @@ def test_add_rtp_status(mcsession, status):
 
 def test_errors_rtp_status(mcsession, status):
     test_session = mcsession
-    pytest.raises(ValueError, test_session.add_rtp_status, 'foo',
-                  *status.status_values[1:])
+    with pytest.raises(ValueError, match="time must be an astropy Time objec"):
```
Comment (plaplant) [inline hera_mc/tests/test_rtp.py:294]: Thanks for making these tests more rigorous 👍

File: hera_mc/tests/test_rtp.py [hunk 2]
```diff
@@ -382,92 +507,292 @@ def test_errors_rtp_process_record(mcsession, observation, record):
...
+        obsid_name: data_obj.task_jobid_columns[obsid_name],
+        "task_name": data_obj.task_jobid_columns['task_name'],
+        "stoptime": data_obj.task_jobid_columns['start_time'] + TimeDelta(5 * 60, format='sec')
+    }
+    result = get_method(**get_keywords)
+    assert len(result) == 1
+    result = result[0]
+    assert result.isclose(expected)
+
+    filename = os.path.join(DATA_PATH, 'test_rtp_task_record_file.csv')
```
Comment (plaplant) [inline hera_mc/tests/test_rtp.py:656]: Do we want to use a temporary directory for this?

File: hera_mc/tests/test_rtp.py [hunk 3]
```diff
@@ -476,95 +801,132 @@ def test_add_rtp_task_resource_record(mcsession, observation, task):
...
+        obsid_name: data_obj.task_resource_columns[obsid_name],
+        "task_name": data_obj.task_resource_columns['task_name'],
+        "stoptime": data_obj.task_resource_columns['start_time'] + TimeDelta(5 * 60, format='sec')
+    }
+    result = get_method(**get_keywords)
     assert len(result) == 1
     result = result[0]
     assert result.isclose(expected)
 
     filename = os.path.join(DATA_PATH, 'test_rtp_task_record_file.csv')
```
Comment (plaplant) [inline hera_mc/tests/test_rtp.py:837]: Maybe also use a temp dir here.

File: hera_mc/tests/test_rtp.py [hunk 4]
```diff
@@ -858,3 +1220,51 @@ def test_get_rtp_launch_record_by_obs_tag(mcsession, observation):
...
+    for ind, result in enumerate(result_task):
+        assert result.isclose(result_obsid_start[ind])
+
+    result_obsid = test_session.get_rtp_task_multiple_track(
+        obsid=multiple_track.track_columns[0]["obsid"]
+    )
+    assert len(result_obsid) == 1
+    assert result_obsid[0].isclose(exp_list[0])
+
+    filename = os.path.join(DATA_PATH, 'test_rtp_track_multiple_file.csv')
```
Comment (plaplant) [inline hera_mc/tests/test_rtp.py:1264]: We can use a temp dir here too.

File: scripts/add_rtp_task_jobid.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2021 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Add RTP task jobid entry to the M&C database with a start_time of "now".
```
Comment (plaplant) [inline scripts/add_rtp_task_jobid.py:6]: Nice descriptive docstring 👍

**Review by plaplant — COMMENTED** [review]
Thanks @bhazelton! This is excellent. I just have a few minor style comments.

File: hera_mc/tests/test_rtp.py (hunk 2 above)
Comment (bhazelton) [reply hera_mc/tests/test_rtp.py:656]: Yes, thanks. I fixed it in all the similar places too.

**Comment by bhazelton** [comment]
Thanks @plaplant! I fixed all the docstrings and I made the tests all use `tmpdir` which is a much better choice.

**Review by plaplant — APPROVED** [review]
Looks great, thanks @bhazelton!

**Pull request merged**